File: Bindings~/java/src/main/java/ro/altom/altunitytester/Notifications/INotificationCallbacks.java
package ro.altom.altunitytester.Notifications;

public interface INotificationCallbacks {
    void SceneLoadedCallBack(AltUnityLoadSceneNotificationResultParams altUnityLoadSceneNotificationResultParams);
    void SceneUnloadedCallBack(String sceneName);
    void LogCallBack(AltUnityLogNotificationResultParams altUnityLogNotificationResultParams);
    void ApplicationPausedCallBack(boolean applicationPaused);
}

File: Assets/AltTester/AltDriver/Notifications/INotificationCallbacks.cs

namespace Altom.AltDriver.Notifications
{
    public interface INotificationCallbacks
    {
        void SceneLoadedCallback(AltLoadSceneNotificationResultParams altUnityLoadSceneNotificationResultParams);
        void SceneUnloadedCallback(string sceneName);
        void LogCallback(AltLogNotificationResultParams altUnityLogNotificationResultParams);
        void ApplicationPausedCallback(bool applicationPaused);
    }
}